agasc/supplement/magnitudes/star_obs_catalogs.py

```diff
@@ -216,28 +225,17 @@ def _load_observed_catalogs():
     ces['col'] = col
 
     STARS_OBS = ces.group_by('agasc_id')
+    STARS_OBS = join(STARS_OBS, _DWELLS, keys=['mp_starcat_time'])
+    STARS_OBS.rename_columns(['tstart', 'tstop'], ['obs_start', 'obs_stop'])
     STARS_OBS.add_index('agasc_id')
     STARS_OBS.add_index('mp_starcat_time')
 
-    # Make a table that contains the indices into `star_obs` and gives the
-    # repeat count `n_obs` for each star.
-
-    indices = STARS_OBS.groups.indices
-    rows = []
-    for idx0, idx1 in zip(indices[:-1], indices[1:]):
-        agasc_id = STARS_OBS['agasc_id'][idx0]
-        rows.append((agasc_id, idx1 - idx0, idx0, idx1))
-    STARS_OBS_INDICES = Table(rows=rows, names=['agasc_id', 'n_obs', 'idx0', 'idx1'])
-    STARS_OBS_INDICES.sort('n_obs')
-    STARS_OBS_INDICES.add_index('agasc_id')
-    # print(stars_obs_indices[:5])
-    # print(stars_obs_indices[-10:])
-
-    STARS_OBS_MAP = {row['agasc_id']: (row['idx0'], row['idx1']) for row in STARS_OBS_INDICES}
-    STARS_OBS_NP = STARS_OBS.as_array()
 
-
-def load(tstop=None):
-    if STARCAT_CMDS is None:
-        _load_startcat_commands(tstop)
-        _load_observed_catalogs()
+def load(tstop=None, commands_v2=False):
+    global STARS_OBS
+    if STARS_OBS is None:
+        if commands_v2:
+            STARS_OBS = get_star_observations(stop=tstop)
+        else:
+            _load_startcat_commands(tstop)
+            _load_observed_catalogs()
```

Comment:
Useful for testing this PR, but at this point all the commands V1 functionality can be removed.

Reply:
this was completely removed.